src/daemon/command_parser_executor.cpp

```diff
@@ -326,11 +326,19 @@ bool t_command_parser_executor::start_mining(const std::vector<std::string>& arg
   
   if(args.size() == 4)
   {
+    if(args[3] != "true" && args[3] != "false")
+    {
+      LOG_PRINT_L0("Incorrect parameter for ignore_battery. Should be true or false.");
```

Comment:
Honestly I wasn't sure if it should show an error and continue since these params are optional or just return false. If you think it would be better to just return false then I'll change it.